packages/ai/core/tool/mcp/mcp-client.ts

```diff
@@ -163,6 +167,25 @@ class MCPClient {
     this.onClose();
   }
 
+  protected assertCapability(method: string): void {
```

Comment:
why `protected` vs `private`/`public`? are there subclasses? is this supposed to be only internally used?

Reply:
Ah! A relic from copying over from MCP's SDK - updating to private